### DISCUSSION TITLE
BUSCANDO TEAM PARA HACKATON(LOS QUE NO TIENEN EQUIPO PODEMOS FORMAR UNO NUEVO :) REPORTARSE AQUI ABAJO  )

### DISCUSSION THREAD

**Discussion by Illescas970103** [Launch-X-Latam/MisionFrontEnd#567] [Team Posts]
Hola team buenas noches, busco equipo para HACKATON, por cuestiones laborales estoy terminando un poquito despues las clases, pero estoy con toda la disposicion de aprender y conocer a todos para participar en el hackaton el cual me emociona mucho.

Si alguien me pudiese considerar este es mi correo.
Mi correo es: lx00645@innovaccion.mx

Gracias buenas noches.

**Comment by oscararcee** [comment]
Hola Alfredo buenos días, yo también estoy buscando equipo para el hackaton, la verdad pensé que ya me habían agregado a uno, pero por lo visto no. Espero de aquí podamos juntar otros colegas para formar el equipo. 

Mi correo es: lx02779@innovaccion.mx 
Saludos.

**Comment by gapman20** [comment]
Yo también ando buscando equipo para el hackaton, si se hace un equipo cuenten conmigo 

mi correo es lx16569@innovaccion.mx

**Comment by CarlosSandoval-03** [comment]
Buen día, me gustaría hacer parte del equipo si aún es posible, en caso tal mi correo es: lx06301@innovaccion.mx

**Comment by oscararcee** [comment]
¿Y como nos agregamos o como esta la cosa?

**Comment by snowingt** [comment]
Me interesa snowingtorres@gmail.com

**Comment by PetuchoDeveloper** [comment]
Hola, me interesa pueden contactarme en Teams
lx15053@innovaccion.mx

**Comment by Crosiux** [comment]
Tambien estoy interesado si gustas te dejo mi correo
lx08113@innovaccion.mx

**Comment by NatoLinko** [comment]
Igual busco formar equipo para el Hackaton,
lx05142@innovaccion.mx

**Comment by dzarkV** [comment]
Hola, yo busco aquipo. Me porto bien.

Mi correo es: lx06352@innovaccion.mx

**Comment by AllGrr** [comment]
Que tal, por si tienen cupo busco equipo para el hachaton mi correo es: lx09923@innovaccion.mx

**Comment by Josdelonnn** [comment]
busco equipo

**Comment by andreleichon24** [comment]
Yo también

**Comment by andreleichon24** [comment]
Les parece si creamos un grupo de Telegram y organizamos grupos?

**Comment by andreleichon24** [comment]
Anexo link del grupo de Telegram
https://t.me/+MrTFx7WJ4IplNGYx

**Comment by Josdelonnn** [comment]
busco equipo

**Comment by Josdelonnn** [comment]
lx06970@innovaccion.mx

**Comment by innovategm** [comment]
aun les falta integrante? 
lx05800@innovaccion.mx

**Comment by Caml02** [comment]
Aqui esta mi correo, por si les interesa tenerme en cuenta-

***@***.***

Saludos!

El mié, 16 nov 2022 a la(s) 19:18, innovategm ***@***.***)
escribió:

> aun les falta integrante?
> ***@***.***
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Launch-X-Latam/teams/mision-backend/discussions/104/comments/17>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVIGTWMAKQD5ZXR7XQQHJ5LWIWIVRANCNFSM6AAAAAASB3W3ZU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 


*Asesor Financiero  *

*Cristian A. Morales L.**_____________________________*
*(662) - 447- 6685*

*Todos Los Bancos En Un Solo Lugar*
